crates/red_knot_workspace/src/workspace/files.rs

```diff
@@ -32,46 +32,67 @@ impl PackageFiles {
         }
     }
 
-    fn indexed(indexed_files: IndexedFiles) -> Self {
+    fn indexed(indexed_files: Arc<FxHashSet<File>>) -> Self {
```

Comment:
It feels kinda awkward how mnay `Arc<FxHashSet<File>>` annotations there are everywhere now. Would it be worth creating a newtype wrapper `struct IndexedFilesInner(Arc<FxHashSet<File>>);` that dereferences to `FxHashSet<File>`? Or at least create a type alias somewhere?

Reply:
A newtype wrapper seems unnecessary complicated. I don't want to hide the type. I can do a type alias, although 4 usages don't seem that many to me and naming now kind of becomes awkward